modules/exploits/linux/games/ut2004_secure.rb

```diff
@@ -89,24 +96,20 @@ def ut_version
   def check
     vers = ut_version
 
-    if (not vers)
-      vprint_status("Could not detect Unreal Tournament Server")
-      return Exploit::CheckCode::Unknown
+    if !vers
+      return CheckCode::Safe('Could not detect Unreal Tournament Server')
```

Comment:
The checkcode here has been changed from `Unknown` to `Safe`. Seems like a manual change and not something rubocop has done. Is there any context on this? 👀

Reply:
changed